rubicon/java/jni.py

```diff
@@ -6,7 +6,15 @@
 
 # If we're on Android, the SO file isn't on the LD_LIBRARY_PATH,
 # so we have to manually specify it using the environment.
-java = cdll.LoadLibrary(os.environ.get('RUBICON_LIBRARY', util.find_library('rubicon')))
+_java_lib = util.find_library('rubicon')
+if _java_lib is None:
+    try:
+        _java_lib = os.environ['RUBICON_LIBRARY']
+    except KeyError:
+        raise ValueError("Can't find Rubicon library")
+
+java = cdll.LoadLibrary(_java_lib)
+
 
 JNI_VERSION_1_1 = 0x00010001
 JNI_VERSION_1_2 = 0x00010002
```

Comment:
I would have set these constants up a while ago... have there been updates for JNI 1.7, 1.8 etc?

Reply:
Beats me :D but I can check!

Reply:
These `JNI_VERSION_1_1` constants were unused, so I have now removed them and replaced them with a comment about how the JNI API has been stable for ages. I think that's better than keeping them and not using them.

Reply:
👍